src/components/dropdown/dropdown.tsx

```diff
@@ -175,18 +176,20 @@ export class Dropdown implements InteractiveComponent, OpenCloseComponent {
   }
 
   render(): VNode {
-    const { active, open } = this;
+    const { active, open, guid } = this;
     return (
       <Host>
         <div
           class="calcite-dropdown-trigger-container"
+          id={`${guid}-menubutton`}
           onClick={this.openCalciteDropdown}
           onKeyDown={this.keyDownHandler}
           ref={this.setReferenceEl}
         >
           <slot
+            aria-controls={`${guid}-menu`}
             aria-expanded={toAriaBoolean(active || open)}
-            aria-haspopup="true"
+            aria-haspopup="menu"
```

Comment:
After testing, it looks like we should use the `listbox` role here to accommodate JAWS in all browsers and NVDA in the Chrome browser (also replaced below).

```html
aria-haspopup="listbox"
```